tlspuffin/src/tls/seeds.rs

```diff
@@ -1300,6 +1122,7 @@ pub fn _seed_client_attacker12(
     (trace, client_verify_data)
 }
 
+// TODO: `Unable to find variable (Some(AgentName(0)), 4)[Some(ApplicationData)]/Message!` error with BoringSSL
```

Comment:
Still the case?

Reply:
I changed this trace to use flights but it seems that no tickets appears in the boringssl trace : boring doesn't send a second flight of packets containing tickets